test/functional-tests/node-transformers/converting-transformers/object-expression-transformer/ObjectExpressionTransformer.spec.ts

```diff
@@ -199,4 +199,37 @@ describe('ObjectExpressionTransformer', () => {
             assert.match(obfuscatedCode, objectSpreadRegExp);
         });
     });
+
+    describe('object spread: unicode escape sequence', () => {
```

Comment:
this test is not covered bug. If following line will removed
```
this.stringsCache.set(`${result}-${String(encodeAllSymbols)}`, result);
```
test still will pass